src/index.js

```diff
@@ -0,0 +1,22 @@
+export const CALL = Symbol("Callable.CALL") // CALL symbol
```

Comment:
> I don't think there's a strong case to export this twice, since it's accessible via `Callable.CALL`.

its just for simplification. Easier to write
[CALL](): {
 do smthing
} 
than
[Callable.CALL]():{
 do smthing
}